packages/connect-core/src/entities/App.ts

```diff
@@ -90,11 +90,11 @@ export default class App {
     return this.artifact.abi
   }
 
-  get intents(): AppIntent[] {
+  get methods(): AppMethod[] {
```

Comment:
I decide to change the name to `methods` instead of `intents` as it was confusing. I'm also wondering if we may want to remove those methods at all 🤔